src/directives/formioElement.js

```diff
@@ -11,9 +11,18 @@ module.exports = [
         element.replaceWith($compile($templateCache.get(scope.template))(scope));
         scope.$emit('formElementRender', element);
       },
-      controller: function() {
-        // This is required for some reason as it will occasionally throw an error without it.
-      }
+      controller: [
+        '$scope',
+        function(
+          $scope
+        ) {
+          $scope.$watchCollection('data.' + $scope.component.key, function(_new, _old) {
```

Comment:
Yeah, it is probably a wash... Let's play around with this approach and change it if it causes problems in the future.